sdk/docs/manually-written/overview/overview/multiparty-application-intro.rst

```diff
@@ -3,180 +3,130 @@
+
+What is a multi-party application?
+----------------------------------
+
+A multi-party application is one in which data, and the rules and workflows that govern the data, are shared between two or more parties without any party having to give up sovereignty or any single party (including the application provider) being able to control or override the agreed rules of the system. A party could be a company, a department within a company, an organization, an individual or a person. The specific definition of a party will be unique to the application and the domain of that application.
+
+A well-designed multi-party application provides several benefits:
+ - a clean, consistent view of all data managed by the application across all parties
+ - consistent, connected, and efficient processes between all parties involved in the application
+ - privacy controls over portions of the shared data, such that each party sees only the data that it is explicitly entitled to view and/or modify
+ - individual party ownership of and responsibility for sensitive data
+
+In most cases, no single party can view all of the data within a multi-party application.
+
+Multi-party applications solve complex operational processes while keeping data clean and consistent, thereby eliminating isolated, disconnected and inefficient processes that often require expensive reconciliation. Multi-party applications manage the relationships, agreements and transactions between parties, providing consistent real-time views of all data.
+
+Multi-party solutions utilize distributed ledger (blockchain) technology to ensure each party has an immutable, consistent view of the shared data and business processes that govern the data. By providing a consistent view of data with all counterparties, a multi-party application removes friction, cost, and risk within a joint business process. A distributed ledger protects against a malicious participant in the network, attempting to write or overwrite data to the detriment of other parties.
+
+Important concepts in multi-party applications
+----------------------------------------------
+
+For a multi-party application to fully deliver its value, the following conditions must be met:
+
+Multiple involved parties have data sovereignty – that is, they keep their data within their own systems and require strong guarantees that no external party can access or modify that data outside of pre-agreed rules.
+Shared state and rules are codified into an executable schema that determines what data can move between parties, who can read that data, and how that data is manipulated.
+Processes happen in real time as there is no additional reconciliation or manual processing required between organizations.
```

Comment:
Consider to use a list for each condition here. Otherwise, it's not clear to me what the conditions really are and how far into this section the list of condition extends.